base_tier_validation_formula/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2019 Creu Blanca
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Base Tier Validation Formula",
+    "summary": """
+        Formulas for Base tier validation""",
+    "version": "17.0.1.0.0",
+    "license": "AGPL-3",
```

Comment:
```suggestion
    "license": "AGPL-3",
    "development_status": "Mature",
```

Reply:
Promotion should'nt have to be in migration commit. But I noticed the missing tag, and it likely satisfies mature status.